---
title: '''Fluxo de trabalho de rastreamento mostra "PGS-220000 [!DNL PostgreSQL] erro: ERRO: erro de relação "t1" não existe"'''''
description: Descrição
solution: Campaign Classic v7
product: Campaign Classic v7
applies-to: Campaign Classic v7
keywords: KCS, Campaign Classic, Workflow de rastreamento, nms:TrackingLog
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Ananya Kuthiala
article-created-date: 5/10/2022 9:07:16 AM
article-published-by: Ananya Kuthiala
article-published-date: 5/10/2022 9:10:25 AM
version-number: 2
article-number: KA-17530
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=68154d91-40d0-ec11-a7b5-0022480a8e40
exl-id: 1a8a9bfd-7dcf-4a7d-b87e-0f17e6e63a08
source-git-commit: 0c3e421beca46d9fe1952b1f98538a50697216a0
workflow-type: tm+mt
source-wordcount: '146'
ht-degree: 2%

---

# O workflow de rastreamento mostra &quot;PGS-220000 [!DNL PostgreSQL] erro: ERRO: erro de relação &quot;t1&quot; não existe&quot;

## Descrição


Após uma atualização de build 9343 ou superior recente, você pode enfrentar os seguintes erros gerados pelo workflow de rastreamento:




```
nlmodule    PGS-220000 PostgreSQL error: ERROR:  relation "t1" does not exist LINE 1: ...t, T3.iArticle, T2.iType FROM NmsTrackingUrl T2,  T1 LEFT OU... 
nlmodule    WDB-200001 SQL statement 'SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2,  T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog  :#(1)# AND T1.tsLog = :#(2)#  AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog' could not be executed.   Param(0)=09/17/2020 8:55:00 PM   Param(1)=09/16/2020 8:55:00 PM
nlmodule    Attribute 'broadLog-id' unknown (see definition of schema 'Tracking logs (nms:trackingLog)').
```





## Resolução


Nos logs de rastreamento detalhados, vemos que o workflow de rastreamento está falhando no final, de consolidar os dados:




```
00000000 00000000 1 info log Consolidating tracking logs for the period 09/16/2020 3:55:00 PM, 09/17/2020 3:55:00 PM...
00000000 00000000 1 info wdbc Select: 00000000  SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog  :#(1)# AND T1.tsLog = :#(2)# AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog
00000000 00000000 2 info wdbc Param(0)=09/17/2020 8:55:00 PM
00000000 00000000 2 info wdbc Param(1)=09/16/2020 8:55:00 PM
00000000 00000000 1 error log PGS-220000 PostgreSQL error: ERROR: relation "t1" does not exist\nLINE 1: ...t, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OU...\n \n. (iRc=-2006)
00000000 00000000 1 error log WDB-200001 SQL statement 'SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog  :#(1)# AND T1.tsLog = :#(2)# AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog' could not be executed.\n Param(0)=09/17/2020 8:55:00 PM\n Param(1)=09/16/2020 8:55:00 PM (iRc=-2006)
```




Toda vez que o workflow de rastreamento é iniciado, ele extrai todos os target mappings e atualiza os dados com base na configuração.

Nesse caso, um target mapping está causando o problema. Consulte abaixo:

![](assets/a06a8deb-6536-ec11-b6e6-000d3a348885.png)

Basicamente,<b> NMS: log de rastreamento</b> O schema NÃO deve ser usado para monitorar logs porque o schema não está relacionado a uma tabela de banco de dados (SQL).

Em conclusão, o problema não tem nada a ver com a atualização, mas mais com uma configuração de target mapping incorreta.
